PersonenOrt.Repository.Service/Controllers/PersonController.cs

```diff
@@ -25,9 +25,25 @@ public IEnumerable<Person> Get()
             }
         }
         [HttpPut("{id:int}")]
-        public Person PutPerson(int id, Person person)
+        public IActionResult PutPerson(int id, Person person)
         {
-            return null;
+            if (person.Id != null)
```

Comment:
Gut und gut... hätte man auch in einer Zeile schreiben können... ist aber ein wenig Geschmackssache...